Bugzilla/Install/Filesystem.pm

```diff
@@ -948,6 +950,13 @@ sub _check_web_server_group {
     return $group_id;
 }
 
+sub robots_txt {
+    my $output = '';
+    Bugzilla->template->process("global/robots.txt.tmpl", {SITEMAP_URL => correct_urlbase().SITEMAP_URL}, \$output) or die Bugzilla->template->error;
```

Comment:
I steered you wrong here. Here's a revised bit of code that add a before_robots_txt hook:
```perl
my %vars;
Bugzilla::Hook::process("before_robots_txt", { vars => \%vars });
Bugzilla->template->process("robots.txt.tmpl", \%vars, \$output)
    or die Bugzilla->template->error;
```